spacy_loggers/wandb.py

```diff
@@ -9,7 +9,6 @@
 from spacy import Language
 from spacy.training.loggers import console_logger
 
-
 # entry point: spacy.WandbLogger.v3
 def wandb_logger_v3(
```

Comment:
We'd want to bump the version to v4 and document the new parameters in the [readme](https://github.com/explosion/spacy-loggers#example-config).

Reply:
It seems to work fine now? I see that you've added it to `[options.entry_points]` as well, which is the main "trick" to get this registered. It might be that your local setup gave test errors, in which case you probably just need to do a quick reinstall from source.

Reply:
I tested the new v4 logger by using it in the wandb integration example project https://github.com/explosion/projects/tree/v3/integrations/wandb 

That worked fine, also correctly versioned models on subsequent runs and everything.

However, doing a straight `pytest` on this repo (in a fresh venv on python 3.9.5) results in the same error as before. Not sure what's missing.

Reply:
I've added the new parameters to the README.

Reply:
> However, doing a straight `pytest` on this repo (in a fresh venv on python 3.9.5) results in the same error as before. Not sure what's missing.

Did you run something like `pip install -e .` first?

Reply:
I did not. Having done that, the tests work fine 👍